lib/vmdb/loggers.rb

```diff
@@ -30,7 +30,10 @@ def self.apply_config(config)
     end
 
     def self.create_logger(log_file_name, logger_class = VMDBLogger)
-      log_file = ManageIQ.root.join("log", log_file_name)
+      log_dir = ManageIQ.root.join("log")
+      log_dir.mkpath unless log_dir.exist?
```

Comment:
I don't think it's that simple.  I tried this on an appliance as root and the directory was created as `root:root` with `755`.  If I `su` to manageiq user, I get `Permission denied`.  It looks like the directory should be `manageiq:manageiq` and `750`.

Reply:
On an appliance I thought the directory was already there.

Reply:
in an appliance the directory should be created by the rpm and should be manageiq:manageiq 4750.

@Fryguy Are you seeing this from a non-rpm environment or from an appliance where rpms were used to install the code? It could be a bug in the rpm spec.